src/components/FullReview.jsx

```diff
@@ -21,18 +22,26 @@ export const FullReview = () => {
     if(reviewLoading) return <h1 className="review-loading-message">Loading Review...</h1>
 
     return(
-        <article className="full-review-container">
-            <h1 className="full-review-title">{fullReview.title}</h1>
-            <h3 className="full-review-category">Category: {fullReview.category}</h3>
-            <img className="full-review-image" src={fullReview.review_img_url} alt="sample-of-board-game-in-action"/>
-            <h3 className="full-review-info"><strong>By: {fullReview.owner}, Votes: {fullReview.votes}, Created At: {fullReview.created_at}</strong></h3>
-            <div className="full-review-body-container">
-                <p className="full-review-body">{fullReview.review_body}</p>
-            </div>
-            <div className="full-review-button-container">
-                <button>❤️</button>
-                <button>Post A Comment</button>
-            </div>
-        </article>
+        <>
+            <article>
+                <section className="full-review-container">
```

Comment:
I assume these changes are for styling as it looks like you've just added a container. Make sure a PR only contains code for the functionality you've implemented. Changes like this can be added on a different branch, or just pushed to main directly.